### PR TITLE
Add the mobile first responsiveness and built the desktop version of my portfolio

### PR DESCRIPTION
**Pull request by mirelapitt**
On this pull request, I built my portfolio desktop version using the best practices of CSS and HTML, as well as the media queries in my CSS file. I also layout the elements on the page using Grid and Flexbox.

### CODE REVIEW

**Review by MrRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @mirelapitt 👋

Good job so far! 👏. Your Coding Style is Impressive! 💪.

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** 🌟 that can improve your code quality 💯 and help you **get a better job** 💼.

_I will guide 📔 you through fixing 🔧 these bugs 🐛, just **follow my comments down below** 🔽 and make sure you **don't miss any** of them, in order to get your project approved ✅_

## Highlights 🌟

- ✔️ descriptive commit messages
- ✔️ very descriptive PR title and description
- ✔️ good effort in code

## Required Changes ♻️

- [ ] [REQUIRED] Please, The Contact Form is not centered. I think It's better to be centered to match the Figma design 🖼️.
  -  image

- [ ] [REQUIRED] Also, make sure that the form in the mobile version is also centerd. In your design, it is shifted to the left.
  - image


_Also Check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code_

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,677 @@
...
+                  type="email"
+                  class="text1"
+                  size="32"
+                  name="contact_email"
+                  maxlength="32"
+                  placeholder="natajayanj14@gmail.com"
+                  required
+                  pattern=".+@\.com"
+                  title="Please Enter Valide Email Id"
+                />
```
Comment (aimemalaika) [inline index.html:659]: - [ ] [OPTIONAL] Kindly correct the email pattern format to allow all king of email address to be used.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,677 @@
...
+                <input
+                  type="text"
+                  class="text1"
+                  name="user_name"
+                  maxlength="30"
+                  placeholder="Yeremias"
+                  pattern="[a-z]{1,30}"
+                  required
+                  title="Username should only contain lowercase letters. e.g John"
+                />
```
Comment (aimemalaika) [inline index.html:646]: - [ ] [OPTIONAL] Kindly allow capital letters in the email field.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+                  title="Please Enter Valide Email Id"
+                />
+              </li>
+              <li>
+                <input
+                  class="msg"
+                  maxlength="500"
+                  name="user_message"
+                  placeholder="Write your message here"
+                />
```
Comment (aimemalaika) [inline index.html:667]: - [ ] [OPTIONAL] Kindly make sure the message field is a `textarea` instead of an input.

**Review by aimemalaika — APPROVED** [review]
Hi @mirelapitt, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**